### DISCUSSION TITLE
Page freezes when I resubmit my benefits application information

### DISCUSSION THREAD

**Discussion by thadryanjs** [community/community#54454] [GitHub Education]
### Hi everyone,

Hello, 

I was resubmitting my application for github student benefits and the page froze when I clicked "continue" after entering my reasons for using github.

Initially, I had declined geolocation permission by accident and thought that might be the issue, so I cleared my cache and tried again. This time I granted access and it froze anyway. 

To make sure it wasn't my browser, I logged in to another browser, logged into to github, and submitted the application again, and it also froze on the new browser. I tried with vivaldi and chrome. 

Refreshing the page clears my info, and let's me re-enter, but it just freezes. I'm left on this screen:

![Screenshot from 2023-05-04 11-40-39](https://user-images.githubusercontent.com/31735051/236258474-e0bb2dd5-d272-4a13-a590-48babb711e42.png)

The page is unresponsive and I can't get out of the re-enter/freeze cycle. 

Please advise!

Thanks!:
    Thad

**Comment by samus-aran** [comment]
Hi @thadryanjs  👋 

Apologies for the delayed response from the team, we hope you were able to resolve your issue. Were you able to successfully submit your application and did you do anything differently that could help others in the community if they too experience a similar situation.

**Comment by thadryanjs** [comment]
Funny timing - I got approved a few days ago. This weekend I tried it again on a whim, and it worked. I was on Firefox. I don't remember which two I tried initially but I just wanted a long time and tried again and got lucky by magic. 

Unfortunately it isn't really reproducible for other community members really, but try it on different wifi networks and different browsers near your school.
